{{cookiecutter.app_name}}/.github/PULL_REQUEST_TEMPLATE/RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Link to the milestone on Github e.g. https://github.com/nimblehq/git-templates/milestone/41?closed=1
+or
+Link to the project management tool for the release
+
+## Features
+
+Provide the ID and title of the issue in the section for each type (feature, chore and bug). The link is optional.
+
+- [ch1234] As a user, I can log in
+  or
+- [[ch1234](https://github.com/nimblehq/git-templates/issues/1234)] As a user, I can log in
+
+## Chores
+- Same structure as in  ## Feature
+
+## Bugs
+- Same structure as in  ## Feature
```

Comment:
```suggestion
## Chores

- Same structure as in  ## Feature

## Bugs

- Same structure as in  ## Feature
```

@longnd Should we add some spaces here?

Reply:
@longnd Can you can this when you have time :D